src/driver/op.rs

```diff
@@ -62,8 +62,8 @@ impl<T> Op<T> {
         F: FnOnce() -> squeue::Entry,
     {
         driver::CURRENT.with(|inner_rc| {
-            let mut inner = inner_rc.borrow_mut();
-            let inner = &mut *inner;
+            let mut inner_ref = inner_rc.borrow_mut();
+            let inner = &mut *inner_ref;
 
             // If the submission queue is full, flush it to the kernel
             if inner.uring.submission().is_full() {
```

Comment:
`inner.submit()?;` at line 70.
Doesn't it meet the same problem?

Reply:
It shouldn't because the `Op` instance hasn't been created yet.